internal/pkg/describe/lb_web_service.go

```diff
@@ -32,11 +32,12 @@ const (
 	envOutputInternalLoadBalancerDNSName = "InternalLoadBalancerDNSName"
```

Comment:
Is this still useful now that we don't display internal ALB DNS name at all.

Reply:
Do we display that in svc show or anywhere currently?

Reply:
yeah it's used in `svc show`, so that would fail sometimes if we removed it